src/ninshiki_cpp_detector.cpp

```diff
@@ -54,56 +55,58 @@ int main(int argc, char ** argv)
 
   // Handle arguments
   try {
-    if (argc < 2) {
-      std::cerr << "Argument needed!\n\n" << help_message << std::endl;
+    if (args.size() < 2) {
+      RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "Argument needed!\n\n" << help_message);
       return 1;
     }
     int i = 1;
     int pos = 0;
-    while (i < argc) {
-      std::string arg = argv[i++];
+    while (i < args.size()) {
+      const std::string& arg = args[i++];
       if (arg[0] == '-') {
         if (arg == "-h" || arg == "--help") {
-          std::cout << help_message << std::endl;
+          RCLCPP_INFO_STREAM(rclcpp::get_logger("ninshiki_cpp"), help_message);
           return 1;
         } else if (arg == "--detector") {
-          std::string value = argv[i++];
+          const std::string& value = args[i++];
           if (value == "yolo") {
             detection_method = value;
           } else {
-            std::cout << "Unknown detector `" << arg << "`!\n\n" << help_message << std::endl;
+            RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "No value provided for `--detector`!\n\n" << help_message);
             return 1;
           }
         } else if (arg == "--GPU") {
-          int value = atoi(argv[i++]);
+          int value = std::stoi(args[i++]);
           if (value == 0 || value == 1) {
             gpu = value;
           } else {
-            std::cout << "Unknown option for GPU `" << arg << "`!\n\n" << help_message << std::endl;
+            RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "No value provided for `--GPU`!\n\n" << help_message);
             return 1;
           }
         } else if (arg == "--MYRIAD") {
-          int value = atoi(argv[i++]);
+          int value = std::stoi(args[i++]);
           if (value == 0 || value == 1) {
             myriad = value;
           } else {
-            std::cout << "Unknown option for MYRIAD `" << arg << "`!\n\n" <<
-              help_message << std::endl;
+            RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "No value provided for `--MYRIAD`!\n\n" << help_message);
             return 1;
           }
         } else if (arg == "--frequency") {
-          frequency = atoi(argv[i++]);
+          frequency = std::stoi(args[i++]);
         } else {
-          std::cout << "Unknown argument `" << arg << "`!\n\n" << help_message << std::endl;
+          RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "Unknown argument `" << arg.c_str() << "`!\n\n" << help_message);
```

Comment:
Shouldn't need to use `c_str()` in here:
```suggestion
          RCLCPP_ERROR_STREAM(rclcpp::get_logger("ninshiki_cpp"), "Unknown argument `" << arg << "`!\n\n" << help_message);
```